lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php

```diff
@@ -105,34 +102,28 @@ public function convertException(string $message, DriverExceptionInterface $exce
                 return new Exception\LockWaitTimeoutException($message, $exception);
             case self::ER_TABLE_EXISTS_ERROR:
                 return new Exception\TableExistsException($message, $exception);
-
```

Comment:
@carusogabriel could you help me understand this too? The new standard introduces new lines around control statements (must be related to https://github.com/slevomat/coding-standard/issues/472) but at the same time removes them from between `case`s.

Instead of removing the newline, could we enforce it instead?

Reply:
It looks like a bug. Reported as https://github.com/slevomat/coding-standard/issues/867.

Reply:
This is related to https://github.com/doctrine/coding-standard/commit/e91edf00502b851bec8b0c930b4ca106d494b13b, but looks like it's something misconfigured rather than a bug

Reply:
I've excluded this rule temporarily until we find the proper solution.